exonum-java-binding/cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/TransferTxHistoryTest.java

```diff
@@ -17,84 +17,80 @@
 
 package com.exonum.binding.cryptocurrency.transactions;
 
-import static com.exonum.binding.cryptocurrency.transactions.ContextUtils.newContextBuilder;
-import static com.exonum.binding.cryptocurrency.transactions.CreateTransferTransactionUtils.createWallet;
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.contains;
+import static com.exonum.binding.cryptocurrency.transactions.TransactionUtils.createCreateWalletTransaction;
+import static com.exonum.binding.cryptocurrency.transactions.TransactionUtils.createTransferTransaction;
+import static org.assertj.core.api.Assertions.assertThat;
 
+import com.exonum.binding.common.crypto.KeyPair;
 import com.exonum.binding.common.crypto.PublicKey;
 import com.exonum.binding.common.hash.HashCode;
+import com.exonum.binding.common.message.TransactionMessage;
 import com.exonum.binding.cryptocurrency.CryptocurrencySchema;
+import com.exonum.binding.cryptocurrency.CryptocurrencyServiceModule;
 import com.exonum.binding.cryptocurrency.PredefinedOwnerKeys;
 import com.exonum.binding.cryptocurrency.Wallet;
-import com.exonum.binding.proxy.Cleaner;
-import com.exonum.binding.storage.database.Database;
-import com.exonum.binding.storage.database.Fork;
-import com.exonum.binding.storage.database.MemoryDb;
 import com.exonum.binding.storage.indices.ProofMapIndexProxy;
 import com.exonum.binding.test.RequiresNativeLibrary;
-import com.exonum.binding.transaction.TransactionContext;
+import com.exonum.binding.testkit.TestKit;
 import com.exonum.binding.util.LibraryLoader;
-import org.hamcrest.Matcher;
 import org.junit.jupiter.api.Test;
 
 @RequiresNativeLibrary
 class TransferTxHistoryTest {
 
+  private static final KeyPair TO_KEY_PAIR = PredefinedOwnerKeys.FIRST_OWNER_KEY_PAIR;
```

Comment:
If bidi transfers are performed in this test, does it make sense to use to/from naming instead of the original?